test/toolchain/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("//rust:toolchain.bzl", "toolchain_tool")
+
+toolchain_tool.cargo(
+    name = "cargo",
+)
+
+toolchain_tool.clippy(
```

Comment:
What is this test testing? Wdyt about adding a test that actually executes these files?

Reply:
I've added more involved tests.